client/state/selectors/get-image-editor-is-greater-than-minimum-dimensions.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { isInteger } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	MinimumImageDimensions
+} from 'state/ui/editor/image-editor/constants';
+import { getImageEditorOriginalAspectRatio } from 'state/selectors';
+
+/**
+ * Returns whether the original image size is greater than minimumImageDimensions values.
+ *
+ * @param  {Object}  state Global state tree
+ * @param   {Integer} minimumWidth the minimum width of the image
+ * @param   {Integer} minimumHeight the minimum height of the image
+ * @returns {Boolean} whether dimensions of the image meet the minimum dimension requirements
+ */
+export default function getImageEditorIsGreaterThanMinimumDimensions(
```

Comment:
Makes sense to me. Only if this particular selector isn't required elsewhere some day.  

I'm not sure  even `getImageEditorOriginalAspectRatio` is utilised anywhere at all. 

But I too will follow the advice of those in know. ;-)